tests/unit-tests/config/__snapshots__/semi-false.test.ts.snap

```diff
@@ -472,10 +472,76 @@ exports[`config > semi: false > it formats ../cases/gts/issue-191-b.gts 1`] = `
 
 exports[`config > semi: false > it formats ../cases/gts/issue-191-c.gts 1`] = `
 "const one = <template>💩💩💩💩💩💩💩</template>
+
```

Comment:
the original code also has a line between the `const two = ...`. I think the behaviour from prettier is to keep the line if one is there